causely/manifest.json

```diff
@@ -0,0 +1,55 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "0ac3b5eb-46d2-4091-a001-e95dc5782609",
+  "app_id": "causely",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Causal AI platform automatically captures, represents and analyzes causality.",
+    "title": "Causely",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "Kubernetes Service Congested Causality",
+        "image_url": "images/Service_Congested_Causality.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Application DB Connection Noisy Neighbor Causality",
+        "image_url": "images/DB_Connections_Causality.png"
+      }
+    ],
+    "classifier_tags": [
+      "Category::Containers",
+      "Category::Kubernetes",
+      "Category::Incidents",
+      "Offering::Integration",
+      "Queried Data Type::Metrics",
+      "Queried Data Type::Events",
+      "Supported OS::Linux",
+      "Supported OS::Windows",
+      "Supported OS::macOS"
+    ]
+  },
+  "assets": {
+    "dashboards": {
+      "Causely Dashboard": "assets/dashboards/causely_dashboard.json"
+    },
+    "integration": {
+      "source_type_name": "Causely",
+      "configuration": {},
+      "events": {
+        "creates_events": true
+      }
+    },
```

Comment:
```suggestion
    }
```

Reply:
sorry for the mistake, thank you for finding it!